lib/rewrite/source/ex.ex

```diff
@@ -344,8 +344,12 @@ defmodule Rewrite.Source.Ex do
     formatter_opts = Keyword.put(formatter_opts, :plugins, plugins)
 
     fn quoted, opts ->
-      opts = update_formatter_opts(formatter_opts, opts)
-      code = Sourceror.to_string(quoted, opts)
+      opts =
+        formatter_opts
+        |> update_formatter_opts(opts)
+        |> eval_deps()
+
+      code = Sourceror.to_string(quoted, Keyword.take(opts, [:locals_without_parens]))
```

Comment:
We should pass all opts to `Sourceror.to_string/2`.
```suggestion
      code = Sourceror.to_string(quoted, opts)
```